python/google/protobuf/internal/message_test.py

```diff
@@ -1638,6 +1638,12 @@ def testMapByteSize(self):
     msg.map_int32_foreign_message[19].c = 128
     self.assertEqual(msg.ByteSize(), size + 1)
 
+  def testMapFieldConstruction(self):
+    msg1 = map_unittest_pb2.TestMap()
+    msg1.map_string_foreign_message['test'].c = 42
+    msg2 = map_unittest_pb2.TestMap(map_string_foreign_message=msg1.map_string_foreign_message)
```

Comment:
Please print this into two lines. Our style guide ask for 80 charactors at most for each line.

Reply:
Do we have similar support for repeated field? I didn't see such test for repeated field
https://github.com/google/protobuf/blob/ed4321d1cb33199984118d801956822842771e7e/python/google/protobuf/internal/reflection_test.py#L150

Reply:
We don't, map fields can't be repeated?

Sorry about the line length, our limit at work is 100 chars and I've gotten used to that :)